api/core/span_context.go

```diff
@@ -24,19 +24,19 @@ type TraceID struct {
 }
 
 const (
-	traceOptionBitMaskSampled = byte(0x01)
-	traceOptionBitMaskUnused  = byte(0xFE)
+	traceFlagsBitMaskSampled = byte(0x01)
+	traceFlagsBitMaskUnused  = byte(0xFE)
 
-	// TraceOptionSampled is a byte with sampled bit set. It is a convenient value initialize
+	// TraceFlagsSampled is a byte with sampled bit set. It is a convenient value initialize
```

Comment:
While you are at it, maybe update this comment, because its grammar generates ENOPARSE in my head. Maybe "TraceFlagsSampled is a byte with sampled bit set. It is a convenient value initializer for SpanContext TraceFlags field when a trace is sampled.".

Reply:
done.